_std/defines/speech_defines/modules.dm

```diff
@@ -91,6 +92,7 @@
 #define LISTEN_INPUT_GHOSTLY_WHISPER "spooky"
 #define LISTEN_INPUT_THRALLCHAT "thrallchat"
 #define LISTEN_INPUT_THRALLCHAT_GLOBAL "global_thrallchat"
+#define LISTEN_INPUT_MENTOR_MOUSE "mentor_mouse"
```

Comment:
All speech defines should be in alphabetical order please.

Reply:
added